### PR TITLE
Epita logo in the javascript console

### PR DESCRIPTION
**Pull request by thisiskeanyvy**
I am reopening the PR following this one : #95

- Displaying the Epita logo encoded in ASCII in the JavaScript console. (script.js)

console

### CODE REVIEW

**Comment by matiboux** [comment]
Copying my comment from the previous PR:

> Why is the ascii background (with @ characters) chunky?
> 
> Wouldn't it be better to have a rectangular filled background with @ characters?
> Or better yet: no background at all, and just use ascii characters for the logo itself.
> What do you think?

To what ChatGPT replied, through you:

> [...] using a rectangular filled background with "@" characters may result in a more visually appealing and consistent display.
> [...] opting for no background and solely using ASCII characters for the logo itself can provide a cleaner and more focused representation.
> 
> [...] the decision on the design and appearance of the [...] logo should consider factors such as readability, aesthetics, and adherence to the overall visual style of Epita.

**Comment by thisiskeanyvy** [comment]
- As I said, I pushed a commit without the background and squashed the commits into one

ascii

**Comment by matiboux** [comment]
As you did not said, you did.

Will review in the following days and merge.

File: assets/js/script.js [hunk 1]
```diff
@@ -93,4 +93,12 @@ $(document).ready(function() {
...
... ####??   ##            #####  (((              \n        ######   #########   ###   ????   #######   ########/   ###             \n    ##########   #########   #???   ???   #######   ########     ###            \n (((((((((((((        ((((   ????   ???   #######   #######   #   ###((#        \n    ((((((((((   ((((((???   ?(    ????   ???????   ??????   ???   ???#####     \n        ((((((   (((??????      ???????   ???????   ?????           ???#######  \n           (((        #???   ??????????   ???????   ????   ???????   ???#####   \n             ??????????????&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&??????       \n          ???? ??????  ?? && & &  &  &&? & & ?& && &  && & ???? ??   ?????      \n       ???????????????&&&&&&&&&&&&&&&&&&&&&&&&&&???????????????????????????     \n                       ???????????????(((                                       \n                      ?????? ((((((((((((                                       \n                                 ((((((((                                       \n                                    ((((((                                      \n                                        ((                                      "));
```
Comment (matiboux) [inline assets/js/script.js:102]: What's the point of `setTimeout`?
The console log function is executed in-place. 
So your code has a type error: `setTimeout` takes `void` as argument.

Also for readability: multi-line string.

```suggestion
       console.log(`
                                   ????                 ((((
                                ????????  (((        ((((((((
                            ???????????? (((((((  ((((((((((((
                        ????????#??####?(((((((((((((((((((((((
                    ####################((((((((((((((((((((((((
                ##########################################(((((((
            ##        (###       ####??   ##            #####  (((
        ######   #########   ###   ????   #######   ########/   ###
    ##########   #########   #???   ???   #######   ########     ###
 (((((((((((((        ((((   ????   ???   #######   #######   #   ###((#
    ((((((((((   ((((((???   ?(    ????   ???????   ??????   ???   ???#####
        ((((((   (((??????      ???????   ???????   ?????           ???#######
           (((        #???   ??????????   ???????   ????   ???????   ???#####
             ??????????????&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&??????
          ???? ??????  ?? && & &  &  &&? & & ?& && &  && & ???? ??   ?????
       ???????????????&&&&&&&&&&&&&&&&&&&&&&&&&&???????????????????????????
                       ???????????????(((
                      ?????? ((((((((((((
                                 ((((((((
                                    ((((((
                                        ((
`)
```

File: assets/js/script.js [hunk 2]
```diff
@@ -93,4 +93,12 @@ $(document).ready(function() {
 		$('#warn-invalid-link').attr({ 'href': href, 'target': target });
 		$('#warn-invalid-modal').modal();
 	});
+
+	// Comment this code to debug the site in the console
+    console.clear()
```
Comment (matiboux) [inline assets/js/script.js:98]: As suggested by the condition on line 99, `console` may be undefined here.
This line of code is stupid.

```suggestion
```

**Review by matiboux — CHANGES_REQUESTED** [review]
Left comments and code suggestions. The code is low quality.

File: assets/js/script.js (hunk 2 above)
Comment (matiboux) [reply assets/js/script.js:98]: Oh and inconsistent indentation as a bonus.

**Comment by thisiskeanyvy** [comment]
.

**Comment by thisiskeanyvy** [comment]
.

**Comment by thisiskeanyvy** [comment]
a

**Comment by thisiskeanyvy** [comment]
b

**Comment by thisiskeanyvy** [comment]
coinbasewallet: {
      package: CoinbaseWalletSDK,
      options: {
        appName: "BuyWithCrypto",
        infuraId: "b54ec21d469a4b0586bce17b6605e643",
        rpc: "",
        chainId: 1,
        darkMode: false
      }
    }

**Pull request closed without merging**